bzt/resources/base-config.yml

```diff
@@ -36,6 +36,8 @@ modules:
     class: bzt.modules.javascript.MochaTester
   nunit:
     class: bzt.modules.csharp.NUnitExecutor
+  xunit:
+    class: bzt.modules.csharp.XUnitExecutor
```

Comment:
This value is basically a fully-qualified name for import. So it should be `bzt.modules.xunitcsharp.XUnitExecutor`.

If you move XUnitExecutor class into `bzt/modules/csharp.py` module - it will become `bzt.modules.csharp.XUnitExecutor`.